### PR TITLE
Re-add nuked files

### PR DESCRIPTION
**Pull request by hello-smile6**
This'll help a lot.

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+custom: https://replit.com/JaydenDev
```
Comment (kccuber-scratch) [inline .github/FUNDING.yml]: We dont need this file

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -12,5 +12,5 @@ jobs:
       - uses: actions/first-interaction@v1
```
Comment (kccuber-scratch) [inline .github/workflows/greetings.yml:12]: meh, i don't see how this is needed

**Review by kccuber-scratch — CHANGES_REQUESTED** [review]

File: .github/workflows/greetings.yml (hunk 2 above)
Comment (kccuber-scratch) [reply .github/workflows/greetings.yml:12]: breh i meant the changes that you made to the file sorry

Comment (hello-smile6) [reply .github/workflows/greetings.yml:12]: Is needed.

Comment (kccuber-scratch) [reply .github/workflows/greetings.yml:12]: nah, it's not really necessary. also I figured out most of these changes so ill just PR these into the main one

Comment (hello-smile6) [reply .github/workflows/greetings.yml:12]: > nah, it's not really necessary. also I figured out most of these changes so ill just PR these into the main one

Okay.

File: .github/FUNDING.yml (hunk 1 above)
Comment (hello-smile6) [reply .github/FUNDING.yml]: Okay.

Comment (hello-smile6) [reply .github/FUNDING.yml]: Fixed. See https://github.com/hello-smile6/freecat/commit/8ffd77122c3ee4e4033547ad63033d527ae829f3

**Comment by hello-smile6** [comment]
Does this work? If so, could you give an approving review?

**Comment by kccuber-scratch** [comment]
> Does this work? If so, could you give an approving review?

eh, i already said im just pushing some of the main changes into the actual freecat

closing, sorry, since i should have made it clear to not PR into main ,and should have made another branch. oops.

**Comment by hello-smile6** [comment]
> > Does this work? If so, could you give an approving review?
> 
> eh, i already said im just pushing some of the main changes into the actual freecat
> 
> closing, sorry, since i should have made it clear to not PR into main ,and should have made another branch. oops.

Okay, what branch should I do?

**Pull request closed without merging**